hasher-matcher-actioner/hmalib/matchers/matchers_base.py

```diff
@@ -58,6 +58,49 @@ def get_privacy_group_matcher_active(privacy_group_id: str) -> bool:
     )
 
 
+@functools.lru_cache(maxsize=128)
+def _get_all_matcher_active_privacy_groups(cache_buster) -> t.List[str]:
+    configs = ThreatExchangeConfig.get_all()
+    return list(
+        map(
+            lambda c: c.name,
+            filter(
+                lambda c: c.matcher_active,
+                configs,
+            ),
+        )
+    )
+
+
+@functools.lru_cache(maxsize=128)
+def _get_all_privacy_group_active_matcher_pdq_theshold(
+    cache_buster,
+) -> t.Dict[str, int]:
+    active_pg_names = _get_all_matcher_active_privacy_groups(cache_buster)
+    if not active_pg_names:
+        return {}
+    return {
```

Comment:
nit: the key of the results dict is never used, only the values. A list would be better unless you plan on doing more filtering later.